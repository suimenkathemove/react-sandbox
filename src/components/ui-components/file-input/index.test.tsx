import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import 'jest-styled-components';
import { FileInput } from './';

const setup = () => render(<FileInput {...defaultProps} />);

describe('FileInput', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
