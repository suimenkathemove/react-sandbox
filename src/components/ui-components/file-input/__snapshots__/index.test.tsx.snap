// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileInput render 1`] = `
<DocumentFragment>
  .c0 {
  display: none;
}

<input
    class="c0"
    type="file"
  />
</DocumentFragment>
`;
